.github/resources/scripts/user_manager.py

```
@@ -0,0 +1,375 @@
+        self.permission_types = self.role_manager.get_permission_types()
+
+    def run_command(self, cmd, description=None, check=True, capture_output=False):
+        """Execute shell command with error handling"""
+        if description:
+            print(f"📋 {description}")
+
+        print(f"🔧 Running: {' '.join(cmd) if isinstance(cmd, list) else cmd}")
+
+        try:
+            if capture_output:
+                result = subprocess.run(cmd, shell=isinstance(cmd, str),
```

Ruff (0.14.13)

`subprocess` call: check for execution of untrusted input

(S603)